lib/kubeclient/common.rb

```diff
@@ -189,8 +189,9 @@ def define_entity_methods
         end
 
         # get a single entity of a specific type by name
-        define_singleton_method("get_#{entity.method_names[0]}") do |name, namespace = nil|
-          get_entity(klass, entity.resource_name, name, namespace)
+        define_singleton_method("get_#{entity.method_names[0]}") \
+            do |name, namespace = nil, opts = {}|
```

Comment:
@NickLaMuro could you forward-port this so that master won't have to change to use `ns` instead of `namespace`?